.travis.yml

```diff
@@ -8,6 +8,7 @@ python:
 
 install:
   - pip install opentracing
+  - pip install git+https://github.com/wavefrontHQ/wavefront-python-sdk.git
```

Comment:
Make sure we replace this once `wavefront-python-sdk` gets published onto PyPI

Reply:
Will change that line once `wavefront-python-sdk` is on PyPI